client/src/Components/Resuable Components/Header/LoggedInNavigationBar.js

```diff
@@ -42,31 +46,72 @@ class LoggedInNavigationBar extends Component {
     ) {
       return <Redirect to={{ pathname: this.state.activeItem }} />;
     }
+
     return (
       <div>
-        <Menu borderless>
-          <Menu.Item
-            name="In A Nutshell"
-            active={this.state.activeItem === "/"}
-            onClick={this.handleItemClick}
-          />
-          <Menu.Item
-            name="Write About a Book"
-            active={this.state.activeItem === "/writeAboutABook"}
-            onClick={this.handleItemClick}
-          />
-          <Menu.Item>
-            <Input icon="search" placeholder="Search..." />
-          </Menu.Item>
+        <Menu borderless secondary>
+          <Responsive as={MenuItem} minWidth={767}>
```

Comment:
Is this minwidth tested on the screen? I think seamntic ui has an option that caters specially towards mobiles, tablets, and large screens right?

Reply:
the mobile/table/large screen stuff works when you have a grid. I couldn't get it to work with responsiveness, e.g. <Responsive only="tablet mobile"/>

the min/maxWidth is used to show or hide things depending on screen widths. 768px is where the smallest iPad starts, so by going 767px, we're just under that screen width to account for mobile phones.

Reply:
okie 👍

Reply:
nvm. it does the same thing as min/maxWidth={767}

Reply:
ooh okie

Reply:
👍  good for pull request bro